### PR TITLE
Adding phpdocumentor

### PR DESCRIPTION
**Pull request by jzarowny**
## What does this PR do?
Add resource on PHP list
### Why is this valuable (or not)?

The goal of the content is to provide knowledge on how to write and subsequently generate effective documentation with phpDocumentor.

### How do we know it's really free?

All the content is available on the link

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -1603,6 +1603,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Clean Code PHP](https://github.com/jupeter/clean-code-php)
 * [PHP 5 Power Programming](http://www.informit.com/content/images/013147149X/downloads/013147149X_book.pdf) (PDF)
 * [PHP Best Practices](https://phpbestpractices.org)
+* [PHP Documentor](https://docs.phpdoc.org/3.0/)
```
Comment (davorpa) [inline books/free-programming-books-langs.md:1606]: ℹ️ Why vrs.3.0? If CONTRIBUTION is read, use latest is out best option.

Link: https://docs.phpdoc.org
Title: `PHP Documentor - Documentation`
Add author `phpdoc.org` to distinguish from other posible resources

**Review by davorpa — CHANGES_REQUESTED** [review]
Please apply suggested changes pushing a new commit

File: books/free-programming-books-langs.md (hunk 1 above)
Comment (jzarowny) [reply books/free-programming-books-langs.md]: I get the wrong URL, thanks.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**